source/Halibut.Tests/BumpThreadPoolForAllTests.cs

```diff
@@ -12,8 +12,11 @@ public class TestsSetupClass
             [OneTimeSetUp]
             public void GlobalSetup()
             {
-                ThreadPool.SetMaxThreads(Int32.MaxValue, Int32.MaxValue);
-                ThreadPool.SetMinThreads(4000, 4000);
+                var minWorkerPoolThreads = 5000;
```

Comment:
I've seen test runs hang locally so something is causing all tests to stop sometimes!!
Try bump these just incase!!